source/LoadPanel.cpp

```diff
@@ -51,35 +51,34 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 using namespace std;
 
 namespace {
-	// Return a pair containing settings to use for time formatting.
-	pair<const char*, const char*> TimestampFormatString(Preferences::DateFormat format)
+	// Return a format string containing settings to use for time formatting.
+	const char * TimestampFormatString(Preferences::DateFormat format)
 	{
-		// pair<string, string>: Linux (1st) and Windows (2nd) format strings.
 		switch(format)
 		{
 			case Preferences::DateFormat::YMD:
-				return make_pair("%F %T", "%F %T");
+				return "%F %T";
 			case Preferences::DateFormat::MDY:
-				return make_pair("%-I:%M %p on %b %-d, %Y", "%#I:%M %p on %b %#d, %Y");
+				return "%I:%M %p on %b %d, %Y";
 			case Preferences::DateFormat::DMY:
 			default:
-				return make_pair("%-I:%M %p on %-d %b %Y", "%#I:%M %p on %#d %b %Y");
+				return "%I:%M %p on %d %b %Y";
 		}
 	}
 
 	// Convert a time_t to a human-readable time and date.
 	string TimestampString(time_t timestamp)
 	{
-		pair<const char*, const char*> format = TimestampFormatString(Preferences::GetDateFormat());
+		const char *format = TimestampFormatString(Preferences::GetDateFormat());
 		stringstream ss;
 
 #ifdef _WIN32
 		tm date;
 		localtime_s(&date, &timestamp);
```

Comment:
I don’t know why this uses localtime_s() and the other one just uses localtime() when they appear to do the same thing, but one just takes the date as a pointer and the other returns it.